src/models/backend/chains.rs

```diff
@@ -28,3 +29,17 @@ pub struct Theme {
     pub text_color: String,
     pub background_color: String,
 }
+
+#[derive(Deserialize, Debug, PartialEq, Clone)]
+#[serde(tag = "type")]
+#[serde(rename_all = "lowercase")]
+pub enum GasPrice {
+    #[serde(rename_all = "camelCase")]
+    Oracle {
+        url: String,
+        gas_parameter: Option<String>,
+        gwei_factor: String,
+    },
+    #[serde(rename_all = "camelCase")]
+    Fixed { wei_value: String },
```

Comment:
```suggestion
    Fixed { wei_value: String },
    #[serde(other)]
    Unknown,
```

Just to protect ourselves from ourselves in the future :D